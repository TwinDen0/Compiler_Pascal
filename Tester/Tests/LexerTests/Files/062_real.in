1.7E2