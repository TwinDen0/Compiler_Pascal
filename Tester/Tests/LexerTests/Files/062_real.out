1	1	REAL	170	1.7e2
EndFile